autoconfigure/README.md

```diff
@@ -0,0 +1,20 @@
+# Configuration
+
+## Broker and topics
+
+| Configuration | Description | Default |
+|---------------|-------------|---------|
+| `KAFKA_BOOTSTRAP_SERVERS` | Kafka bootstrap servers | `localhost:9092` |
+| `KAFKA_SPAN_TOPIC` | Topic where incoming spans are stored. | `zipkin-span` |
+| `KAFKA_TRACE_TOPIC` | Topic where aggregated traces are stored. | `zipkin-trace` |
+| `KAFKA_DEPENDENCY_TOPIC` | Topic where aggregated service dependencies names are stored. | `zipkin-dependency` |
+
+## Storage configurations
+
+| Configuration | Description | Default |
+|---------------|-------------|---------|
+| `KAFKA_STORAGE_DIR` | Root path where Zipkin stores tracing data | `/data` |
+| `KAFKA_STORAGE_TRACE_RETENTION_SCAN_FREQUENCY` | How often to validate traces retention. This will be evaluated at the time events are received. (ms) | `3600000` (1 hour) |
```

Comment:
Please also simplify the TRACE_RETENTION_SCAN_FREQUENCY to something else and consider making it an internally documented setting unless you see people turning this knob frequently. This is more about garbage collection iiuc

Reply:
Removed from `README` and renamed to `TRACE_GC_INTERVAL`